### PR TITLE
Adding Docker Info and training materials sources

### PR DESCRIPTION
**Pull request by simplymanas**
more to follow

### CODE REVIEW

**Comment by ChaituVR** [comment]
Hey @simplymanas 

So I'd love to be able to merge your changes but it looks like you have some merge conflicts. ⚠️

Once you resolve these conflicts, I will be able to review your PR and merge it. 😊

---

> If you're not familiar with the merge conflict process, feel free to look over GitHub's guide on ["Resolving a merge conflict"](https://help.github.com/articles/resolving-a-merge-conflict-on-github/). 🔍️
>
> Also, it's good practice on GitHub to write a brief description of your changes when creating a PR. 📝

**Comment by BerkeleyTrue** [comment]
Deploy preview failed.

Built with commit d6f934118fba16cc7f0ceddfe1a160110923837c

https://app.netlify.com/sites/guide/deploys/5a3161c18198766e712399b5

**Comment by simplymanas** [comment]
@ChaituVR  can you check now, if still there are issues

File: src/pages/developer-tools/docker/index.md [hunk 1]
```diff
@@ -12,10 +12,57 @@ Similar to Github, DockerHub offers a wide variety of images (Ubuntu and others)
...
 You can find plenty of informations in the following sites : 
 - <a href='https://www.docker.com/' target='_blank' rel='nofollow'>Docker website</a>
 - <a href='https://docs.docker.com/' target='_blank' rel='nofollow'>Docker docs</a>
 - <a href='https://hub.docker.com/' target='_blank' rel='nofollow'>DockerHub</a>
+[logo]: https://www.docker.com/sites/default/files/Whale%20Logo332_5.png "Docker"
+
+
+## Docker
+
+Docker is an open source project to automate the dployment of any software applications inside containers.  A container image is a lightwieght, stand-alone, execuateble package of a piece of software that includes everything need to run it.
```
Comment (ChaituVR) [inline src/pages/developer-tools/docker/index.md]: Description for the Docker already exists can you compare both and make it better? 

and move the content to its correct positions ??

File: src/pages/developer-tools/docker/index.md [hunk 2]
```diff
@@ -12,10 +12,57 @@ Similar to Github, DockerHub offers a wide variety of images (Ubuntu and others)
...
+This message shows that your installation appears to be working correctly.
+
+
+### Where to use Docker
+
+* [8 Proven Real-World Ways to Use Docker](https://www.airpair.com/docker/posts/8-proven-real-world-ways-to-use-docker)
+
+* [What is Docker and When to Use It](https://www.ctl.io/developers/blog/post/what-is-docker-and-when-to-use-it/)
+
+### Docker Tutorials
```
Comment (ChaituVR) [inline src/pages/developer-tools/docker/index.md]: move these links under 

`#### More Information:` 

and make these as anchor links

**Comment by ChaituVR** [comment]
Sorry for the Delay, Thanks for the PR! :+1:

**Pull request merged**